src/Resources/config/routing/admin.yaml

```diff
@@ -0,0 +1,6 @@
+bitbag_sylius_banner_plugin_banners_ad:
```

Comment:
I think that there will be so few endpoints here that instead of dividing each one into separate files, it is better to do it per admin and shop